CHANGELOG.asciidoc

```diff
@@ -3,6 +3,66 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-8.16.6]]
+=== Beats version 8.16.6
+https://github.com/elastic/beats/compare/v8.16.5\...v8.16.6[View commits]
+
+==== Breaking changes
+
+*Affecting all Beats*
+
+- The Beats logger and file output rotate files when necessary. The beat now forces a file rotation when unexpectedly writing to a file through a symbolic link.
+
+*Filebeat*
+
+- Fixed race conditions in the global ratelimit processor that could drop events or apply rate limiting incorrectly.
+
+*Functionbeat*
+
+- Removal of Functionbeat binaries from CI pipelines. {issue}40745[40745] {pull}43152[43152]
```

Comment:
This line disappeared from https://github.com/elastic/beats/pull/43432, which I think is correct as this isn't a user facing change it's an internal one. Possibly notable though that we don't test Functionbeat at all but it's been deprecated for some time already.

Reply:
Good catch, forgot to remove it from here too.